.github/workflows/pr-ci.yml

```
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CloudFormation Python Plugin CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
```

Should we [start testing](https://github.com/actions/python-versions/blob/ac42875cb8afbc2ce400bf6db60105d6ea4c50f0/versions-manifest.json#L95) for [Python 3.9](https://docs.python.org/release/3.9.0/whatsnew/3.9.html) as well?
```suggestion
        python: [3.6, 3.7, 3.8, 3.9]
```